src/ui/components/KeyboardShortcuts.tsx

```diff
@@ -111,15 +112,25 @@ function KeyboardShortcuts({
       toggleQuickOpen(query, project);
     };
 
-    const addComment = (e: KeyboardEvent) => {
+    const addComment = async (e: KeyboardEvent) => {
       // Un-authenticated users can't comment on Replays.
       if (!isAuthenticated) {
         return;
       }
 
       if (!e.target || !isEditableElement(e.target)) {
         e.preventDefault();
-        createFrameComment(null, recordingId);
+
+        let base64PNG: string | null = null;
+        const canvas = document.querySelector("canvas#graphics");
+        if (canvas) {
+          base64PNG = await getBase64Png(canvas as HTMLCanvasElement, {
+            maxWidth: 200,
+            maxHeight: 200,
+          });
```

Comment:
The exact max dimension is debatable. We can't send the full image (too large, GraphQL server will barf)

Reply:
Alternately we could take an image of the area around the marker.